Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -486,6 +492,24 @@ static void webkit_website_data_manager_class_init(WebKitWebsiteDataManagerClass
             nullptr, nullptr,
             FALSE,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY)));
+
+    /**
+     * WebKitWebsiteDataManager:per-origin-storage-quota:
+     *
+     * Max size of origin data storage.
+     * Value of 0 means no change so whatever the default value is it will be used.
```

Comment:
We should document the units, is this size in bytes? KB? MB?

Reply:
It is in bytes (the same as defaultStandardReportedQuota: https://github.com/WebKit/WebKit/blob/main/Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp#L72)